hoomd/md/module-md.cc

```diff
@@ -375,7 +373,6 @@ PYBIND11_MODULE(_md, m)
         m,
         "PotentialSpecialPairCoulombGPU");
     export_BondTablePotentialGPU(m);
-    export_TablePotentialGPU(m);
```

Comment:
When this is working for the GPU we will need to export that to Python as well.

Reply:
I believe this is exporting the old `Table `potential implementation

Reply:
It is; I meant that we will need to expose the Table potential on the GPU too, similar to the other pair potentials.